bundler/spec/dependabot/bundler/file_updater_spec.rb

```diff
@@ -378,10 +380,6 @@
             expect(file.content).to include("statesman (1.3.1)")
           end
 
-          it "locks the yanked gem to the latest version allowed by the Gemfile", :bundler_v1_only do
-            expect(file.content).to include("business (1.18.0)")
-          end
-
           it "does not touch the yanked gem", :bundler_v2_only do
```

Comment:
Again, what's this `:bundler_v2_only`? Is it still relevant?

Reply:
bundler_v2_only is completely removed. Specs are going to work only once for bundler